coral/src/domain/topic/topic-api.msw.ts

```diff
@@ -35,13 +33,7 @@ function mockTopicGetRequest({
 
       // Passing an response will the precedence over scenario.
```

Comment:
Probably the comment needs to be edited, as it references `scenario`, which doesn't exist any more.